src/odemis/dataio/csv.py

```diff
@@ -62,13 +63,26 @@ def export(filename, data):
             csv_writer = csv.writer(fd)
             csv_writer.writerow(headers)
             csv_writer.writerows(spectrum_tuples)
-    elif data.ndim == 2 and all(s >= 2 for s in data.shape):
+    elif data.ndim == 2 and all(s >= 2 for s in data.shape) and (data.metadata.get(model.MD_DESCRIPTION, None) == "Angle-resolved"):
         logging.debug("Exporting AR data to CSV")
-        # Data should be in the form of (Y+1, X+1), with the first row and colum the angles
+        # Data should be in the form of (Y+1, X+1), with the first row and column the angles
         with open(filename, 'w') as fd:
             csv_writer = csv.writer(fd)
             # Set the 'header' in the 0,0 element
             first_row = ['theta\phi(rad)'] + [d for d in data[0, 1:]]
             csv_writer.writerow(first_row)
             # dump the array
             csv_writer.writerows(data[1:, :])
+    elif data.ndim == 2:
+        # FIXME: For now it handles the rest of 2d data as spectrum-line
+        logging.debug("Exporting spectrum-line data to CSV")
+        # Data should be in the form of (Y+1, X+1), with the first row and column the
+        # distance_from_origin\wavelength
+        with open(filename, 'w') as fd:
+            csv_writer = csv.writer(fd)
+            # Set the 'header' in the 0,0 element
+            first_row = ['distance_from_origin(m)\wavelength(m)'] + [d for d in data[0, 1:]]
```

Comment:
For point spectrum data we check if MD_WL_LIST or MD_WL_POLYNOMIAL metadata is there to decide if unit is "m" or "px". However the line spectrum data only contains pixel size metadata, so it would always give "px". Should I copy the wavelength metadata from spectrum point to spectrum line when I am about to export?